zson/ztests/container-types.yaml

```diff
@@ -0,0 +1,22 @@
+script: |
+  zq -t -i zson "by typeof(.) | sort ." in.zson
```

Comment:
Now that #2123 has been merged, can this be a zql test instead of a script test?

Reply:
it can, done. (looks like there's a number of pre-existing tests that can be similarly changed).